schema/trigger/timingtriggercandidatemaker.jsonnet

```diff
@@ -6,6 +6,7 @@ local types = {
 	time_t : s.number("time_t", "i8", doc="Time"),
 	signal_type_t : s.number("signal_type_t", "u4", doc="Signal type"),
 	hsi_tt_pt : s.boolean("hsi_tt_pt"),
+        count_t : s.number("count_t", "i8", doc="Counter"),
```

Comment:
Does i8 mean 8-bit int? If so, then it allows for negative prescales (which wouldn't make sense, so u8 would be better), and only maximal prescale of 127 (so I would change it to uint16, which would allow for max of 65535, or even uint32).
Might be wrong how i8 is defined though, and how it works with the python -> cpp interface...

Reply:
I'm not sure and I agree and share your scepticism about this. 
My reasoning was this:
in daqconf version of this schema (https://github.com/DUNE-DAQ/daqconf/blob/develop/schema/daqconf/triggergen.jsonnet) there already existed a type for counting `types.count` which is defined here: https://github.com/DUNE-DAQ/daqconf/blob/mrigan/ttcm_prescale/schema/daqconf/types.jsonnet#L21C3-L21C75 as `i8`. I simply propagated this type to trigger as well.

Reply:
This type `count` is also used for many other variables that are definitely just supposed to be positive ints (like `trigger_window_before_ticks`)

Reply:
so options are:
- leave as is, since this is already widely used
- create a new type for this counting
- limit this specific type `count_t` to have a minimum of 1
Let me know

Reply:
Ah, thank you for this. I can see [HERE](https://github.com/DUNE-DAQ/daqconf/blob/940cce5a98aaabf44f1589b94885369faa2a842b/schema/daqconf/types.jsonnet#L11-L16) that these numbers are defined as bytes, no bits. So i8 is actually standard 64-bit int, i4 would be 32-bit int etc. Makes sense, thank you! I still think u8 would be better than i8 since we cannot have negative pre-scale, but I don't think it matters that much.

Reply:
You added more just before I pressed "Comment" :) Unless the other options are literally 5-min of work,  I would say leave as is for now (so your option 1)

Reply:
Thanks for the comments. I have now added a check that ensures the value is >=1. 
Otherwise, this error message is shown:
```ValueError: illegal count_t number 0 not greater than or equal 1```

Also confirmed previous tests still work as expected.

Reply:
Thank you, in that case let's merge the PR :)